### PR TITLE
feat: front end projects config

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

config for https://github.com/freeCodeCamp/demo-projects/pull/322

Closes #XXXXX

### CODE REVIEW

**Comment by ShaunSHamilton** [comment]
Thank you for taking the time to contribute, _random person_.

Please familiarise yourself with our [Contributor Docs](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request) on how to open a PR. Specifically, how to give PRs titles.

File: configs/upstreams.conf [hunk 1]
```diff
@@ -130,3 +130,23 @@ upstream technical-documentation-page {
 upstream tribute-page {
   server localhost:50270;
 }
+
+upstream 25--5-clock {
+  server localhost:50005;
+}
+
+upstream drum-machine {
+  server localhost:50030;
```
Comment (ShaunSHamilton) [inline configs/upstreams.conf]:
```suggestion
  server localhost:50038;
```

File: configs/upstreams.conf [hunk 2]
```diff
@@ -130,3 +130,23 @@ upstream technical-documentation-page {
...
+}
+
+upstream javascript-calculator {
+  server localhost:50075;
+}
+
+upstream markdown-previewer {
+  server localhost:50075;
```
Comment (ShaunSHamilton) [inline configs/upstreams.conf]:
```suggestion
  server localhost:50205;
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
To match the port-map

File: configs/upstreams.conf (hunk 1 above)
Comment (majestic-owl448) [reply configs/upstreams.conf]: I was doing with the port file open on the side 😓😓

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 

I assume this has to go in first?

**Review by moT01 — APPROVED** [review]
LGTM @ieahleen 🎉

**Pull request merged**